posthog/models/cohort.py

```
@@ -109,6 +114,42 @@ def calculate_people(self, use_clickhouse=is_ee_enabled()):
+                sql, params = persons_query.distinct("pk").only("pk").query.sql_with_params()
+                query = UPDATE_QUERY.format(
+                    cohort_id=self.pk,
+                    values_query=sql.replace('FROM "posthog_person"', ', {} FROM "posthog_person"'.format(self.pk), 1,),
+                )
+                cursor.execute(query, params)
+
+            self.is_calculating = False
+            self.last_calculation = timezone.now()
+            self.errors_calculating = 0
+            self.save()
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)